.goreleaser.yaml

```
@@ -13,7 +14,8 @@ universal_binaries:
     name_template: oasis
 
 env:
-  - CGO_ENABLED=1
+  - CGO_ENABLED=0
```

This should remain the same.

```suggestion
  - CGO_ENABLED=1
```